models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md

```diff
@@ -87,4 +87,4 @@ python3 <omz_dir>/tools/downloader/converter.py --name <model_name>
 
 The original model is distributed under the
 [Apache License, Version 2.0](https://raw.githubusercontent.com/rwightman/gen-efficientnet-pytorch/5e91628ed98250989a7ddd20abfe27385e0493c1/LICENSE).
-A copy of the license is provided in [APACHE-2.0-PyTorch-EfficientNet.txt](../licenses/APACHE-2.0-PyTorch-EfficientNet.txt).
+A copy of the license is provided in [APACHE-2.0-PyTorch-Models.txt](../licenses/APACHE-2.0-PyTorch-Models.txt).
```

Comment:
on my view more correct naming is PyTorch-Image-Models like basic repo name for nfnets, efficientnet and so on
https://github.com/rwightman/pytorch-image-models

Reply:
Done